### PR TITLE
Strata - UI: playwright e2e test

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity/issues/25087

*Description of changes:*

https://github.com/user-attachments/assets/d76e3bc8-47fe-4fb5-8d65-92620b54f135

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the BC Registry and Digital Services BSD 3-Clause License

### CODE REVIEW

File: strr-platform-web/playwright.config.ts
Comment (deetz99) [inline strr-platform-web/playwright.config.ts:1]: lint only

File: strr-platform-web/tests/unit/utils/isoCountriesList.test.ts
Comment (deetz99) [inline strr-platform-web/tests/unit/utils/isoCountriesList.test.ts:1]: util in base layer, dont need to test in platform dir

File: strr-platform-web/package.json
Comment (deetz99) [inline strr-platform-web/package.json:1]: test was failing in ui mode, updating is passing now

File: strr-strata-web/app/pages/default.vue
Comment (deetz99) [inline strr-strata-web/app/pages/default.vue:1]: do not need index page

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-examiner-dev--pr-422-yx226r9a.web.app

File: strr-strata-web/app/pages/strata-hotel/application.vue
Comment (deetz99) [inline strr-strata-web/app/pages/strata-hotel/application.vue:1]: this was causing an issue with vue router and resulted in a 404 in the playwright test, moved page to match file based routing instead

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-422-bjoo3jpc.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-422-mid0lghz.web.app

File: strr-strata-web/tests/e2e/strata-smoke.test.ts
Comment (deetz99) [inline strr-strata-web/tests/e2e/strata-smoke.test.ts:1]: main test file

File: strr-strata-web/tests/fake-files/fake-strata-hotel-docs.pdf
Comment (deetz99) [inline strr-strata-web/tests/fake-files/fake-strata-hotel-docs.pdf:1]: empty pdf to use in test file uploader

**Review by thorwolpert — APPROVED** [review]

**Review by ser-popov — APPROVED** [review]
Looks great .. amazing work as always @deetz99

**Pull request merged**